src/puppetlabs/trapperkeeper/authorization/ring_middleware.clj

```diff
@@ -209,11 +209,14 @@
    allow-header-cert-info :- schema/Bool
    oid-map :- acl/OIDMap]
   (if-let [cert (request->cert request allow-header-cert-info)]
-    (let [extensions (ssl-utils/get-extensions cert)
+    (let [oid-map (merge acl/default-oid-map oid-map)
```

Comment:
@nathanielksmith Yeah, I understand where you're coming from on that.  I still think I'd be more nervous about missing the application of the default oids if we have to do it in multiple places in various utility code rather than just in one place at the top of the incoming injection from the middleware API functions.

@KevinCorcoran - what do you think about this?  Would you have a specific preference?